pykokkos/lib/ufuncs.py

```diff
@@ -3009,3 +3009,307 @@ def isfinite(view):
                             view=view,
                             out=out)
     return out
+
+@pk.workunit
+def round_impl_1d_double(tid: int, view: pk.View1D[pk.double], out: pk.View1D[pk.double]):
+    out[tid] = round(view[tid])
+
+
+@pk.workunit
+def round_impl_2d_double(tid: int, view: pk.View2D[pk.double], out: pk.View2D[pk.double]):
+    for i in range(view.extent(1)):
+        out[tid][i] = round(view[tid][i])
+
+
+@pk.workunit
+def round_impl_3d_double(tid: int, view: pk.View3D[pk.double], out: pk.View3D[pk.double]):
+    for i in range(view.extent(1)):
+        for j in range(view.extent(2)):
+            out[tid][i][j] = round(view[tid][i][j])
+
+@pk.workunit
+def round_impl_1d_float(tid: int, view: pk.View1D[pk.float], out: pk.View1D[pk.float]):
+    out[tid] = round(view[tid])
+
+
+@pk.workunit
+def round_impl_2d_float(tid: int, view: pk.View2D[pk.float], out: pk.View2D[pk.float]):
+    for i in range(view.extent(1)):
+        out[tid][i] = round(view[tid][i])
+
+
+@pk.workunit
+def round_impl_3d_float(tid: int, view: pk.View3D[pk.float], out: pk.View3D[pk.float]):
+    for i in range(view.extent(1)):
+        for j in range(view.extent(2)):
+            out[tid][i][j] = round(view[tid][i][j])
```

Comment:
I don't think this will work, see https://github.com/kokkos/pykokkos/blob/a39282171630803067c8c2f13858e0a9beae2eb9/pykokkos/core/visitors/visitors_util.py#L70 for a list of functions that get mapped to their `cmath` counterparts

Reply:
Actually there is a `round` in cmath, it's just not part of this set. Could you add it here as well?